### PR TITLE
style(components): rename disabled to is-disabled

### PR DESCRIPTION
**Pull request by freedomlang**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/pagination/src/components/pager.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <ul :class="nsPager.b()" @click="onPagerClick" @keyup.enter="onEnter">
     <li
       v-if="pageCount > 0"
-      :class="{ active: currentPage === 1, disabled }"
+      :class="[{ active: currentPage === 1 }, nsPager.is('disabled', disabled)]"
```
Comment (JeremyWuuuuu) [inline packages/components/pagination/src/components/pager.vue:5]: **Praise:** It's good to pay this debt.

Comment (JeremyWuuuuu) [reply packages/components/pagination/src/components/pager.vue:5]: **Question:** Could you update `active` as well? Like this:

```ts
[ 
  nsPager.is('active', currentPage === 1),
  nsPager.is('disabled', disabled)
]
```

File: packages/components/pagination/src/components/pager.vue [hunk 2]
```diff
@@ -21,7 +26,10 @@
     <li
       v-for="pager in pagers"
       :key="pager"
-      :class="{ active: currentPage === pager, disabled }"
+      :class="[
+        { active: currentPage === pager },
```
Comment (JeremyWuuuuu) [inline packages/components/pagination/src/components/pager.vue:30]: Same here.

**Review by jw-foss — COMMENTED** [review]
@freedomlang I left a few comments for you to consider. Let me know if you have any questions

**Comment by JeremyWuuuuu** [comment]
@freedomlang please update the test cases accordingly, the tests were failing because we changed the class name and the selector were not selecting any element.
image

**Review by jw-foss — APPROVED** [review]

**Pull request merged**